framework/src/samplers/Sampler.C

```diff
@@ -86,3 +112,30 @@ Sampler::setNumberOfRequiedRandomSeeds(const std::size_t & number)
 
   _generator.saveState();
 }
+
+void
+Sampler::setSampleNames(std::vector<std::string> names)
+{
+  _sample_names = names;
+  mooseAssert(getSamples().size() == _sample_names.size(),
+              "The number of sample names must match the number of samples returned.");
+}
+
+Sampler::Location
+Sampler::getLocation(unsigned int global_index)
+{
+  if (_offsets.empty())
+    reinit(getSamples());
+
+  std::vector<unsigned int>::const_iterator iter;
+  iter = std::upper_bound(_offsets.begin(), _offsets.end(), global_index) - 1;
```

Comment:
Normally, one would do:
```
std::vector<unsigned int>::const_iterator iter = std::upper_bound(_offsets.begin(), _offsets.end(), global_index) - 1;
```
i.e. on a single line. But, I am assuming clang-format will not keep it on a single line. So, was that a reason to have this on 2 lines?